merlin/models/tf/outputs/contrastive.py

```diff
@@ -84,6 +84,20 @@ class ContrastiveOutput(ModelOutput):
     and word classifiers: A loss framework for language modeling. arXiv preprint
     arXiv:1611.01462 (2016).
 
+    Notes:
+    ----------
+    In case `to_call` is set as `DotProduct()`, schema of target couldn't be inferred
+    therefore, the user should feed a schema only with ITEM_ID feature as schema arg,
+    which is treated as a `kwargs` arg below.
+
+    Example usage::
+        outputs=mm.ContrastiveOutput(
+            to_call=DotProduct(),
+            negative_samplers="in-batch",
+            schema=schema.select_by_tag(Tags.ITEM_ID),
+            post=post_logits,
+            logits_temperature = 0.2,
+        )
```

Comment:
We can maybe remove the argument `post=post_logits` from the example as it is not defined and might be confusing to the user. 
We could also add an example where the `schema` parameter is not needed (this is the case where the target schema is passed as the `to_call` arg, for example)
```
outputs = mm.ContrastiveOutput(
    to_call = schema.select_by_tag(Tags.ITEM_ID),
    negative_samplers="in-batch",
    logits_temperature = 0.2,
)
```

Reply:
oh you are right. I forgot to remove that :)